test/unit/storage-component.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { AuthIdentity } from '@dcl/crypto'
+import { createStorageComponent } from '../../src/ports/storage/component'
+import { IStorageComponent } from '../../src/ports/storage/types'
+import { createTestIdentity, generateRandomIdentityId } from '../utils/test-identity'
+
+describe('when testing storage component', () => {
+  let storage: IStorageComponent
+  let validAuthIdentity: AuthIdentity
+  let identityId: string
+
+  beforeEach(async () => {
+    storage = createStorageComponent()
+
+    identityId = generateRandomIdentityId()
+
+    const testIdentity = await createTestIdentity()
+    validAuthIdentity = testIdentity.authChain
+  })
+
+  describe('when storing identity IDs', () => {
```

Comment:
Would you mind renaming the scopes? They seem to have the old naming of the identities storage.
```suggestion
  describe('when storing identity', () => {
```